mage_ai/orchestration/db/migrations/versions/90d978a8aef8_update_unique_constraint_for_secret.py

```diff
@@ -20,11 +20,17 @@ def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
     try:
         bind = op.get_bind()
-        insp = sa.inspect(bind.engine)
-        with op.batch_alter_table('secret', schema=None) as batch_op:
-            unique_constraints = insp.get_unique_constraints('secret')
-            for constraint in unique_constraints:
-                batch_op.drop_constraint(constraint['name'], type_='unique')
+        insp = sa.inspect(bind)
```

Comment:
Changing this line affects the `insp` in the `else` block as well.

Reply:
Yes, but it did not affect the results when I tested with different sql servers. 

SQL server inspection did not work if I had the original line `sa.inspect(bind.engine)`. All inspections calls freeze the migration if bind.engine is used with sql server. I read the documentation the inspect method does accept either an engine or a connection which the bind variable is https://docs.sqlalchemy.org/en/20/core/inspection.html#available-inspection-targets .

Reply:
Here is also documentation about the get_bind() function that states that it returns a connection when "online" mode https://alembic.sqlalchemy.org/en/latest/api/runtime.html#alembic.runtime.environment.EnvironmentContext.get_bind